translate/storage/po.py

```diff
@@ -36,13 +36,10 @@
 
 if platform.python_implementation() == "CPython":
     if usecpo == "1":
-        logging.info("Using cPO")
         from translate.storage.cpo import *  # pylint: disable=W0401,W0614
     elif usecpo == "2":
-        logging.info("Using new fPO")
         from translate.storage.fpo import *  # pylint: disable=W0401,W0614
     else:
-        logging.info("Using Python PO")
```

Comment:
Testing the new Pootle command to list contributors I did finally saw why you want to drop this. Anyway I think it is better to use `logging.debug`.